ui/components/multichain/notification-detail-button/notification-detail-button.tsx

```diff
@@ -43,9 +44,10 @@ export const NotificationDetailButton = ({
       properties: {
         notification_id: notification.id,
         notification_type: notification.type,
-        ...('chain_id' in notification && {
-          chain_id: notification.chain_id,
-        }),
+        ...(notification.type !== TRIGGER_TYPES.FEATURES_ANNOUNCEMENT &&
```

Comment:
Seems like the pervious condition for this was simpler?

Reply:
yeah true, I may have been getting a type error at some point, but it's good now.